Packages/eu.netherlands3d.twin-features/Scripts/EnableComponentsByFeature.cs

```diff
@@ -7,25 +7,6 @@ public class EnableComponentsByFeature : MonoBehaviour
     {
         public List<FeatureLink> FeatureLinks = new();
 
-        private void OnValidate()
```

Comment:
Geen leesbare namen meer in de inspector? Krijg je dan niet dat het overzicht onduidelijk wordt?

Reply:
Eens, alleen dit werkte niet meer omdat er feitelijk meerdere dingen kunnen hangen aan zo'n event.
Zet de OnValidate terug en vervang het nu met:

- 1 listener? toon gameobject naam
- meerdere listeners? toon aantal listeners